<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cryptor - A Cryptography Program</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="hash_sel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hash Function:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>69</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MD5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SHA-1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="encrypt_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hash_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="encrypt">
     <property name="text">
      <string>Encryption Type:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="key_type">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Asymmetric - RSA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Symmetric - DES</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Symmetric - AES</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="hash_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3r23f2f</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::NoEcho</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Genearate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QLabel" name="hash_lab">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hashes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../background-blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="cipher_lab">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ciphers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>29</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_sel">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>From files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_sel">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>From files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_box">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>80</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_key">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>170</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>progressBar</zorder>
   <zorder>hash_sel</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>encrypt_button</zorder>
   <zorder>decrypt_button</zorder>
   <zorder>hash_input</zorder>
   <zorder>splitter</zorder>
   <zorder>splitter_3</zorder>
   <zorder>splitter_2</zorder>
   <zorder>hash_output</zorder>
   <zorder>status</zorder>
   <zorder>gen_button</zorder>
   <zorder>copy_button</zorder>
   <zorder>hash_lab</zorder>
   <zorder>cipher_lab</zorder>
   <zorder>key</zorder>
   <zorder>input</zorder>
   <zorder>file_sel</zorder>
   <zorder>key_sel</zorder>
   <zorder>input_box</zorder>
   <zorder>input_key</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
